homeassistant/package_constraints.txt

```diff
@@ -141,10 +141,6 @@ pubnub!=6.4.0
 # https://github.com/dahlia/iso4217/issues/16
 iso4217!=1.10.20220401
 
-# Matplotlib 3.6.2 has issues building wheels on armhf/armv7
```

Comment:
Can we now build the latest matplotlib wheel on armhf/armv7?

Reply:
Unfortunately I don't have access to an armhf machine to test this. Upstream has arm64 wheels and right now 2.6.1 (pinned) doesn't work even on x86-64 with HA and Python 3.12 so I guess the change would be an improvement to the current situation either way.

Reply:
I checked with Franck and the problem at the time was building wheels pycocotools, needed by tensorflow. We don't ship with the tensorflow integration in our container distributions anymore and have disabled building wheels for pycocotools. The conclusion is that we can remove the pinning of matplotlib here. :+1: